index.ts

```diff
@@ -1,13 +1,35 @@
 import { routes as GetRoutes } from './src/modules/controllers/get.crud'
 import { routes as PostRoutes } from './src/modules/controllers/post.crud'
+import {LocalArrayRepository} from "./src/modules/infraestructure/repositories/custom/local.array.repository";
+import {Service} from "./src/modules/services/services.crud";
 const { fastifyAwilixPlugin } = require('fastify-awilix')
+const { diContainer } = require('fastify-awilix')
+const { asClass, asFunction, Lifetime } = require('awilix')
 
 // Require the framework and instantiate it
 const fastify = require('fastify')({
   logger: true
 })
 fastify.register(fastifyAwilixPlugin, { disposeOnClose: true, disposeOnResponse: true })
 
+diContainer.register({
```

Comment:
I have used the hook because I have pretended to instantiate the dependency by request entity param and be able to differentiate the requests to /users and instantiate the specific service but for now we still no need it (right now, entities are not differentiated in the generic repository, everything goes to the same dataset)